### ISSUE TITLE
Icons task fails on first run-through

### ISSUE THREAD

**Issue by hesselberg** [Novicell/novicell-frontend#52]
When you clone a clean install of the frontend package, you get the following error when running `gulp`. I'm guessing we need to make sure, it doesn't depend on the `icons` folder in the `dist` folder?

```
{ Error: ENOENT: no such file or directory, mkdir './dist/icons/'
    at Error (native)
    at Object.fs.mkdirSync (fs.js:922:18)
    at writeFile (/Users/larshesselberg/Arbejde/Beneath/Kunder/Kilroy/00000 KILROY - Brand Portal 2016-2018/Benns Frontend/gulp/tasks/icons.js:20:16)
    at genereateIconJsonLibrary (/Users/larshesselberg/Arbejde/Beneath/Kunder/Kilroy/00000 KILROY - Brand Portal 2016-2018/Benns Frontend/gulp/tasks/icons.js:66:9)
    at FSReqWrap.oncomplete (fs.js:123:15)
  errno: -2,
  code: 'ENOENT',
  syscall: 'mkdir',
  path: './dist/icons/' }
```

**Comment by marklonquist** [comment]
It seems the issue is with how 'fs' creates directories. It does not create them recursively, so if ./dist does not exist, it can't create the icons folder.

Will change creation of folders to a recursive method.

**Comment by Dan9boi** [comment]
Was fixed in PR #53
